src/main/kotlin/controllers/AbstractSettingsDialog.kt

```diff
@@ -0,0 +1,188 @@
+package controllers
```

Comment:
Note: might want to diff this file with old SettingsController since a lot was taken from there